components/org.apache.stratos.autoscaler/src/main/java/org/apache/stratos/autoscaler/applications/parser/DefaultApplicationParser.java

```diff
@@ -378,13 +374,13 @@ private Application buildCompositeAppStructure(ApplicationContext applicationCon
 
             // get hostname and cluster id
             ClusterInformation clusterInfo;
-            if (cartridgeInfo.getMultiTenant()) {
+            if (cartridge.getMultiTenant()) {
```

Comment:
In the Cartridge class, the getter setter methods are isMultiTenant and setMultiTenant. In DefaultApplicationParser class, we are using the Cartridge from the Stub, where the getMultiTenant method is generated from MultiTenant attribute.